covid_graphs/covid_graphs/formula_test.py

```diff
@@ -0,0 +1,36 @@
+import datetime
+import math
+
+import numpy as np
+
+from .country_report import CountryReport
+from .formula import AtgFormula
+
+
+def test_two_curves():
+    cumulative_active = np.array([0, 0, 1, 2, 5, 18, 45])
+    start_date = datetime.date(2020, 4, 17)
+    dates = [start_date + datetime.timedelta(days=d) for d in range(len(cumulative_active))]
+    report = CountryReport("UK", "United Kingdom", dates, None, None, None, None, cumulative_active)
```

Comment:
I'd suggest to use the named params here.

Reply:
Done